packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -444,7 +444,7 @@ function RichTextWrapper(
 
 			if (
 				__unstableEmbedURLOnPaste &&
-				isEmpty( value ) &&
+				value?.text?.length === 0 &&
```

Comment:
The `isEmpty` function has a similar code to this line, do we need to explicitly check that `value` and `value.text` are defined?

https://github.com/WordPress/gutenberg/blob/6731534775328df6373aed2ca60f41fc642fcc5d/packages/rich-text/src/is-empty.js#L8-L18

Reply:
Good catch! Resolved in 00a4f9d